src/cookie-control-config.ts

```diff
@@ -77,6 +77,14 @@ export const cookieControlConfig: CookieControlConfig = {
 					event: "cookie.analytics.revoke",
 					cookieStorage: "none",
 				});
+
+				window.localStorage.removeItem("vwoSn");
+				for (let i = 0; i < window.localStorage.length; i++) {
+					const item = window.localStorage.key(i) || "";
+					if (item?.match(/_hj.*/)?.length > 0) {
```

Comment:
I think the regex `/^_hj/` is better as it matches the start - that's what we're interested in.

Also we can use [RegExp.prototype.test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) here instead of match. `test` returns a boolean, and is a better option when we don't care about what's match, only that is is. It then becomes simpler:

```suggestion
					if (/^_hj/.test(key)) {
```